server.go

```diff
@@ -740,6 +740,17 @@ func (server *Server) loadConfig(configurations configs, globalConfiguration Glo
 								negroni.Use(metricsMiddlewareBackend)
 							}
 						}
+
+						if len(frontend.WhitelistSourceRange) > 0 {
+							log.Info("Configured IP Whitelists: ", frontend.WhitelistSourceRange)
```

Comment:
I'd place this log messages only after we are clear there won't be an error returned by `NewIPWhitelister`.